src/hooks/useDapps/useDapps.ts

```diff
@@ -169,7 +169,9 @@ export default function useDapps({ useStorage, fetch }: UseDappsProps): UseDapps
             match = categoryFilter.filter(item, favorites)
           }
           if (search && match) {
-            match = item.name.toLowerCase().includes(search?.toLowerCase())
+            const matchedName = item.name.toLowerCase().includes(search?.toLowerCase())
+            const matchedDescription = item?.description.toLowerCase().includes(search?.toLowerCase())
```

Comment:
@PetromirDev looking into the above line `item.name....` it looks like that item is alwasys an object. Having this in mind, shouldn't we use the optional chaining operator after the `.description` field, as follows `item.description?.toLowerCase()`

Currently it's working, because all the dApps are having description field, but it would be better to refactor it.